apps/web/src/composables/walletConnect.ts

```diff
@@ -0,0 +1,84 @@
+import WalletConnect from '@walletconnect/client'
+import QRCodeModal from '@walletconnect/qrcode-modal'
+import { ref, Ref } from 'vue'
+import { ethers } from 'ethers'
+
+export default function useWalletConnect() {
+  const connector: Ref<WalletConnect | undefined> = ref()
+  const walletConnectAddress: Ref<string> = ref('')
+  function enableWalletConnect() {
+    connector.value = new WalletConnect({
+      bridge: 'https://bridge.walletconnect.org', // Required
+      qrcodeModal: QRCodeModal,
+    })
+    if (!connector.value.connected) {
+      connector.value.createSession()
+    }
+    connector.value.on('connect', (error: any, payload: any) => {
+      if (error) {
+        throw error
+      }
+      // Get provided accounts and chainId
+      const { accounts, chainId } = payload.params[0]
+      walletConnectAddress.value = accounts[0]
+    })
+    connector.value.on('session_update', (error: any, payload: any) => {
+      if (error) {
+        throw error
+      }
+      // Get updated accounts and chainId
+      const { accounts, chainId } = payload.params[0]
+    })
+    connector.value.on('disconnect', (error: any) => {
+      if (error) {
+        console.log(`disconnect error :>> ${error}`)
+        // throw error
+      }
+      // Delete connector
+      try {
+        connector.value?.killSession()
+      } catch (error) {
+        console.log(`disconnect error in listener :>> ${error}`)
+      }
+    })
+    return
+  }
+
+  async function sendWalletConnectTx(amount: string, toAddress: string) {
```

Comment:
Let's consistently use either `Transaction` or `Tx`. Looks like ethers uses the former, and we have `sendIoPayTransaction`, so maybe let's go with that.